maps/glloydstation/glloydstation.dm

```diff
@@ -1,5 +1,8 @@
 #if !defined(USING_MAP_DATUM)
 
+	#include "../../code/modules/urist/area&turf/areas.dm"
+	#include "../../code/modules/urist/machinery/misc.dm"
```

Comment:
put this file back where it was. Move the camera definition in that file to glloydstation_presets .dm